### PR TITLE
19908 & 19907 - PUB/SUB Queue upgrades for Python 3.8 master branch

### PR DESCRIPTION
**Pull request by seeker25**
https://github.com/bcgov/entity/issues/19908
https://github.com/bcgov/entity/issues/19907

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: queue_services/account-mailer/src/account_mailer/resources/worker.py [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+    if not (event_message := queue.get_simple_cloud_event(request)):
+        # Return a 200, so event is removed from the Queue
+        return {}, HTTPStatus.OK
+
+    try:
+        message_type, email_msg = event_message.type, event_message.data
+        current_app.logger.debug('message_type received %s', message_type)
+        email_msg['logo_url'] = minio_service.MinioService.get_minio_public_url('bc_logo_for_email.png')
+
+        handle_drawdown_request(message_type, email_msg)
```
Comment (ochiu) [inline queue_services/account-mailer/src/account_mailer/resources/worker.py:50]: Way easier to read and find things!

**Review by ochiu — COMMENTED** [review]
Amazing job on all the clean up so far! thanks!

Will give it another pass when it is ready.

**Comment by seeker25** [comment]
Note: Will refactor enums into sbc-common-components. Still waiting on Jia to provide the enums for pay.

**Comment by seeker25** [comment]
Haven't done much local testing, just rebuilt new queues - unit tests based off of requests. 
Still need to configure GCP PUB/SUB and make sure all secrets are in line etc.

**Review by ochiu — APPROVED** [review]

**Comment by seeker25** [comment]
I've setup the topics in GCP for DEV and updated the diagrams. 

Next: testing auth-api/auth-queue/account-mailer locally

**Comment by seeker25** [comment]
Seems to be working locally using ngrok.

**Comment by seeker25** [comment]
Works locally with Namex-pay. Some small tweaks were required. I've let Andriy know.

**Review by Jxio — APPROVED** [review]

**Review by pwei1018 — APPROVED** [review]

**Pull request merged**